charts/redpanda/values.schema.json

```
@@ -1552,6 +1552,32 @@
           }
         }
       }
+    },
+    "auditLogging": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean"
+        },
+        "listener": {
+          "type": "string"
+        },
+        "partitions": {
+          "type": "integer"
+        },
+        "logReplicationFactor": {
+          "type": "integer"
+        },
+        "enabledEventTypes": {
+          "type": ["array", "null"]
+        },
+        "excludedTopics": {
+          "type": ["array", "null"]
+        },
+        "excludedPrincipals": {
+          "type": ["array", "null"]
+        }
+      }
```

adding in remaining cluster configs

```suggestion
    "auditLogging": {
      "type": "object",
      "properties": {
        "enabled": {
          "type": "boolean"
        },
        "listener": {
          "type": "string"
        },
        "partitions": {
          "type": "integer"
        },
        "logReplicationFactor": {
          "type": ["integer", "null"]
        },
        "enabledEventTypes": {
          "type": ["array", "null"]
        },
        "excludedTopics": {
          "type": ["array", "null"]
        },
        "excludedPrincipals": {
          "type": ["array", "null"]
        },
        "audit_client_max_buffer_size": {
          "type": "integer"
        },
        "audit_queue_drain_interval_ms": {
          "type": "integer"
        },
        "audit_queue_max_buffer_size_per_shard": {
          "type": "integer"
        }
      }
```

---

Are you saying you want these additional items exposed to the customer. The schema is just to validate the values file.